src/htss/plans/tomography.py

```diff
@@ -69,23 +70,36 @@ def tomography_scan(
     num_darks = num_darks or num_aux_images
     num_flats = num_flats or num_aux_images
 
-    for movable in (x, theta, beam):
-        if isinstance(movable, Readable):
-            detectors.append(movable)
-
-    metadata = {
-        "detectors": [det.name for det in detectors],
-        "min_theta": min_theta,
-        "max_theta": max_theta,
-        "num_projections": num_projections,
-        "num_darks": num_darks,
-        "num_flats": num_flats,
-        "x_start": x_start,
-        "x_stop": x_stop,
-        "x_steps": x_steps,
-        "out_of_beam": out_of_beam,
-        **(metadata or {}),
+    md = {
```

Comment:
I think the metadata argument of the function should become md and this _md, just to be consistent with what we see from Bluesky